### ISSUE TITLE
Lazy Torpedo & Sinking Torpedo

### ISSUE THREAD

**Issue by Raldeme** [msummers123/Movecraft-3#246]
### The Lazy Torpedo Issue
So I have came across an issue on my server where every so often the LaunchTorpedo sign will hiccup and turn my torpedo lazy, it successfully pilots it according to the chat, however the torpedo will not move and becomes unmovable until I restart the server, here is the error I am getting. http://pastebin.com/dZ0UtNqu

(P.S.) somehow im running version 5.5.0 ?!?! no idea where i got it as only 5.0.2 is available, running spigot 1.11.2 also this isnt happening with every torpedo, mainly just 1 ship in 1 area, however other ships in same area are not having this issue, the only difference i can see is that it has a dual torpedo launch (2 torps 1 sign) which I have had working in earlier versions of movecraft. and also it doesn't happen with every use, i managed to fire 10 rounds of double torpedo before it spat an error out at me while testing for this issue report, and then sometimes it'd give me the error straight away after restarting 3-4 times, then back to normal.
 
### The Sinking Torpedo issue
Basically after the first block of my torpedo hits (2 block torp) , it tells me the craft is damaged and is going to sink, then the rest of the torpedo sinks without doing anything.

**Comment by TheRealBotcrusher** [comment]
What you are seeing for the sinking torpedo is a change in how torpedo
behavior is treated.

The later blocks do fall off, especially on longer torpedoes, however their
presence is now being counted more during the initial blast, which can be
noted if you compare a single block torpedo to a seven block long torpedo.

I'm not sure as to the cause of the Lazy Torpedo bug though.

**Comment by Raldeme** [comment]
ah with the sinking torp issue, in older versions the torp would just keep on moving and blast its way through so im not used to seeing the back end of a torpedo sinking after it hits its target

**Comment by TylerS1066** [comment]
The stuck torp is because movecraft starts detecting two separate torps from the same sign and has issues, I would suggest a different design.

**Comment by pistols12** [comment]
I personally just use the older movecraft version to where this change was not made, I like the old style torpedos, but eventually will have to change myself.
